packaging/debian/control

```diff
@@ -1,14 +1,14 @@
-Source: ansible
+Source: ansible-base
 Section: admin
 Priority: optional
 Standards-Version: 3.9.3
 Maintainer: Ansible, Inc. <info@ansible.com>
-Build-Depends: cdbs, debhelper (>= 5.0.0), python-docutils, python, dh-python | python-support, python-setuptools, lsb-release
+Build-Depends: cdbs, debhelper (>= 5.0.0), python3-docutils, python, dh-python | python-support, python3-setuptools, lsb-release
 Homepage: https://github.com/ansible/ansible/
 
-Package: ansible
+Package: ansible-base
 Architecture: all
-Depends: python-jinja2, python-yaml, python-paramiko, python-httplib2, python-six, python-crypto (>= 2.6), python-setuptools, sshpass, ${misc:Depends}, ${python:Depends}
+Depends: python3-jinja2, python3-yaml, python3-paramiko, python3-httplib2, python3-six, python3-crypto (>= 2.6), python3-setuptools, sshpass, ${misc:Depends}, ${python:Depends}
```

Comment:
I think we can drop httplib2, setuptools, and crypto here and add cryptography

Reply:
Do we also want to drop paramiko? It was removed from our requirements.txt over a year ago.

Reply:
We do still depend on paramiko for the RHEL packages so that seems like something to keep.

Reply:
Do we need `python3-six`? I thought we vendored that.

+1 to removing `httplib2` and `setuptools`, and replacing `crypto` with `cryptography`